ui/app/components/auth-form.js

```diff
@@ -219,6 +219,8 @@ export default Component.extend(DEFAULTS, {
             };
           })
         );
+        // without unloading the records there will be an issue where all methods set to list when unauthenticated will appear for all namespaces
```

Comment:
Is this because of the upgrade? I thought since we sent the namespace as the header, the `/sys/internal/ui/mounts` requests only fires for that namespace

Reply:
Right, but we have to unload the store here or else that call will just append to the store we already have. This note is just to clarify what the below is doing, so that we can remove it later (`unloadAll` was causing issues throughout tests)

Reply:
Ahh I see - I thought the store cleared with the new request. Thanks for clarifying!